drf_client/connection.py

```diff
@@ -311,6 +312,14 @@ def login(self, password, username=None):
 
         payload = json.dumps(data)
         r = requests.post(url, data=payload, headers=DEFAULT_HEADERS)
+
+        if r.status_code == 401:
+            logger.info('This server requires multi-factor authentication:')
+            otp = getpass.getpass('MFA:')
+            otp_token = r.json()['otp_token']
+            otp_data = {'otp_token': otp_token, 'otp': otp}
```

Comment:
We cannot hard code these two strings. You need to add this to the options and use `self.options["OTP_TOKEN_KEY"` and `self.options["OTP_KEY"]` or something like that